phpstan-baseline.neon

```diff
@@ -10,3 +10,9 @@ parameters:
 			count: 1
 			path: src/Integration/Symfony/Bundle/src/DependencyInjection/Configuration.php
 
+		-   "#^PHPDoc tag @param has invalid value#"
```

Comment:
phpstan don't like the Enum param docblock.. I had to ignore all those rules :/

Reply:
Im okey with this. Pslam is happy.